bin/inv_wrapper.sh

```diff
@@ -3,4 +3,4 @@ set -e
 
 # Wrapper script for running invoke in virtual env
 
-source bin/workon.sh && inv -r faasmcli/faasmcli $@
+source bin/workon.sh && inv -r faasmcli/faasmcli "$@"
```

Comment:
Added quotes, as otherwise if we pass a command with quotes they would not be preserved.